File: api-server/src/main/java/com/example/proyectoaplicaciones/api_server/post/Post.java
package com.example.proyectoaplicaciones.api_server.post;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;

@Data
@Entity
public class Post {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private String title;
    private String body;
    private int score;
    private int userId;
}

File: app/src/main/java/com/example/proyectoaplicaciones/Data/Model/Post.kt
package com.example.proyectoaplicaciones.Data.Model

//Esto es para el tema de la validacion y que funcione en el backend
data class Post(
    val userId: Int,
    val id: Int,
    val title: String,
    val body: String,
    var score: Int = 0 // El único contador de popularidad
)
